src/accessibility/context_menu/ContextMenuTooltipButton.tsx

```diff
@@ -25,6 +25,7 @@ interface IProps extends React.ComponentProps<typeof AccessibleTooltipButton>  {
     isExpanded: boolean;
 }
 
+// TODO: replace this the header buttons and the right panel buttons with a single representation
```

Comment:
grammar? Also not sure what the header buttons and right panel buttons have with a11y context menu buttons?